crates/typst-pdf/src/lib.rs

```diff
@@ -147,16 +151,34 @@ pub enum Timezone {
 /// Encapsulates a list of compatible PDF standards.
 #[derive(Clone)]
 pub struct PdfStandards {
-    /// For now, we simplify to just PDF/A, since we only support PDF/A-2b. But
-    /// it can be more fine-grained in the future.
+    /// For now, we simplify to just PDF/A. But it can be more fine-grained in
+    /// the future.
     pub(crate) pdfa: bool,
+    /// Whether the standard allows for embedding any kind of file into the PDF.
+    /// We disallow this for PDF/A-2, since it only allows embedding other
+    /// PDF/A-2 documents.
```

Comment:
A comment in `write_embedded_files` mentions that an embedding in PDF/A-2 might be PDF/A-2 or PDF/A-1. Here we say it needs to be A-2.
Do I understand the spec correctly that any PDF/A file would be allowed as embedding in a PDF/A-2 file?

Reply:
I didn't check myself, but @reknih told me that PDF/A-1 or PDF/A-2 is fine. I didn't see this comment here, so it should be updated.